tests/cbmc/proofs/s2n_stuffer_alloc/Makefile

```diff
@@ -11,20 +11,24 @@
 # implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-###########
-#Use the default set of CBMC flags
+# Expected runtime is 20 seconds.
+
 CHECKFLAGS +=
 
 DEPENDENCIES += $(HELPERDIR)/source/make_common_datastructures.c
 DEPENDENCIES += $(HELPERDIR)/source/proof_allocators.c
 DEPENDENCIES += $(SRCDIR)/stuffer/s2n_stuffer.c
 DEPENDENCIES += $(SRCDIR)/utils/s2n_blob.c
 DEPENDENCIES += $(SRCDIR)/utils/s2n_ensure.c
+DEPENDENCIES += $(SRCDIR)/utils/s2n_mem.c
+DEPENDENCIES += $(SRCDIR)/utils/s2n_safety.c
 
 ENTRY = s2n_stuffer_alloc_harness
 
-#No loops to unwind
+# We abstract these functions because manual inspection demonstrates they are unreachable.
+REMOVE_FUNCTION_BODY += --remove-function-body s2n_blob_slice
+REMOVE_FUNCTION_BODY += --remove-function-body s2n_ensure_memcpy_trace
```

Comment:
Why was this showing up in the report if its unreachable?

Reply:
`s2n_realloc` is called with more restricted arguments (filtered by previous calls), so these functions are never called, but they decrease our coverage.

Reply:
We can document this by adding a stub for those functions in the harness whose body is `assert(false)`.

Reply:
@danielsn This would still decreases our coverage, since this function will never be called. I think the new `viewer` has a mechanism to avoid this. Should I add an issue so we track this down when we upgrade?